manim/scene/scene.py

```diff
@@ -887,6 +731,30 @@ def get_wait_time_progression(self, duration, stop_condition):
             )
         return time_progression
 
+    def get_run_time(self, animations):
+        """
+        Gets the total run time for a list of animations.
+
+        Parameters
+        ----------
+        animations: list of Animation
+            A list of the animations whose total
+            run_time is to be calculated.
+
+        Returns
+        ------
+        float
+            The total run_time of all of the animations in the list.
+        """
+
+        return np.max([animation.run_time for animation in animations])
+
+    def play(self, *args, **kwargs):
+        self.renderer.play(self, *args, **kwargs)
+
+    def wait(self, duration=DEFAULT_WAIT_TIME, stop_condition=None):
```

Comment:
Suggestion : add a depreciation warning

Reply:
I don't think this should ever be deprecated, just used as more convenient syntax.